main.js

```diff
@@ -364,6 +364,16 @@ function windowCreate(json) {
         sessionCreate(elements[json.targetID].webContents, json.sessionId)
         client.write(json.targetID, consts.eventNames.windowEventDidFinishLoad)
     })
+    elements[json.targetID].webContents.on('did-get-redirect-request', (event, oldUrl, newUrl) => {
+        client.write(json.targetID, consts.eventNames.windowEventDidGetRedirectRequest, {
+            url: newUrl
+        })
+    })
+    elements[json.targetID].webContents.on('will-navigate', (event, url) => {
+        client.write(json.targetID, consts.eventNames.windowEventWillNavigate, {
+            url
```

Comment:
I'm not familiar with that notation. Does writing `url` on its own means the payload will be the following (if the value of url was `https://github.com`): `{"url":"https://github.com"}` ?

Reply:
the `{ url }` will parse to `{ "url" : url }`